contracts/balancer/IndexPool.sol

```diff
@@ -135,11 +77,8 @@ contract IndexPool is BToken, BMath, IIndexPool {
   // requisite initial balance.
   mapping(address => uint256) internal _minimumBalances;
 
-  // Maximum LP tokens that can be bound.
-  // Used in alpha to restrict the economic impact of a catastrophic
-  // failure. It can be gradually increased as the pool continues to
-  // not be exploited.
-  uint256 internal _maxPoolTokens;
+  // Recipient for exit fees
+  address internal _exitFeeRecipient;
```

Comment:
Is this an upgrade? Would this not be a corrupted field here?

Reply:
The value is currently 0 as this feature is not being used, so it'll be interpreted as a null address.